clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/deploy/description/servergroup/DeployKubernetesAtomicOperationDescription.groovy

```diff
@@ -113,6 +114,28 @@ class KubernetesContainerDescription {
   KubernetesSecurityContext securityContext
 }
 
+@AutoClone
+@Canonical
+class KubernetesInitContainerDescription {
```

Comment:
The the above answer is 'yes', maybe we can reflect that here by having the regular container definition inherit from here.

Reply:
Yeah, that would be good. Do you think having something like this would okay?

```
class KubernetesBaseContainerDescription {
...
}

class KubernetesInitContainerDescription extends KubernetesBaseContainerDescription{}
class KubernetesContainerDescription extends KubernetesBaseContainerDescription{
// extra properies
}
```

Reply:
or maybe the extra layer isn't necessary...